widget-reference/modal.md

```diff
@@ -0,0 +1,19 @@
+---
+description: >-
+  Modal is a simple UI widget you can use when you want to create Dialogs, Popovers or Alerts.
+---
+
+# Modal
+
+![Click to expand](../.gitbook/assets/modal.gif)
+
+| Property | Description |
+| :--- | :--- |
+| **Quick Dismiss** | Quickly dismisses or closes the Modal when the user taps outside the modal. |
```

Comment:
@marks0351 can you mention opening a modal using the showModal function and referencing that function page